stix_shifter_modules/rhacs/stix_translation/json/stix_2_1/to_stix_map.json

```diff
@@ -0,0 +1,226 @@
+    }
+  ],
+  "clusterId": {
+    "key": "x-rhacs-cluster.cluster_id",
+    "object": "cluster"
+  },
+  "namespace": {
+    "key": "x-rhacs-cluster.namespace",
+    "object": "cluster"
+  },
+  "namespaceId": {
+    "key": "x-rhacs-cluster.namespace_id",
+    "object": "cluster"
+  },
+  "deploymentId": {
+    "key": "x-rhacs-deployment.deployment_id",
+    "object": "deployment"
+  },
+  "deployment": [
+    {
+      "key": "x-rhacs-deployment.deployment_name",
+      "object": "deployment"
+    },
+    {
+      "key": "x-ibm-finding.extensions.x-rhacs-finding-ext.deployment_refs",
+      "object": "finding",
+      "references": [
+        "deployment"
+      ]
+    }
+  ],
+  "lifecycleStage": {
+    "key": "x-ibm-finding.extensions.x-rhacs-finding-ext.lifecycle_stage",
+    "object": "finding"
+  },
+  "policyId": [
+    {
+      "key": "x-rhacs-policy.policy_id",
+      "object": "policy"
+    },
+    {
+      "key": "x-ibm-finding.extensions.x-rhacs-finding-ext.policy_refs",
+      "object": "finding",
+      "references": [
+        "policy"
+      ]
+    }
+  ],
+  "description": {
+    "key": "x-rhacs-policy.description",
+    "object": "policy"
+  },
+  "rationale": {
+    "key": "x-rhacs-policy.rationale",
+    "object": "policy"
+  },
+  "remediation": {
+    "key": "x-rhacs-policy.remediation",
+    "object": "policy"
+  },
+  "disabled": {
+    "key": "x-rhacs-policy.disabled",
+    "object": "policy"
+  },
+  "categories": {
+    "key": "x-rhacs-policy.categories",
+    "object": "policy"
+  },
+  "eventSource": {
+    "key": "x-rhacs-policy.event_source",
+    "object": "policy"
+  },
+  "severity": {
+    "key": "x-ibm-finding.severity-ex",
+    "object": "finding",
+    "transformer": "SeverityToScore"
+  },
+  "lastUpdated": {
+    "key": "x-rhacs-policy.last_updated",
+    "object": "policy"
+  },
+  "sortName": {
+    "key": "x-rhacs-policy.sort_name",
+    "object": "policy"
+  },
+  "sortLifecycleStage": {
+    "key": "x-rhacs-policy.sort_lifecycle_stage",
+    "object": "policy"
+  },
+  "violationMessage": {
+    "key": "x-ibm-finding.extensions.x-rhacs-finding-ext.violation_message",
+    "object": "finding"
+  },
+  "violationState": {
+    "key": "x-ibm-finding.extensions.x-rhacs-finding-ext.state",
+    "object": "finding"
+  },
+  "process": {
+    "name": [
+      {
+        "key": "process.name",
+        "object": "proc"
+      },
+      {
+        "key": "x-rhacs-container.process_ref",
+        "object": "cntr",
+        "references": "proc"
+      }
+    ],
+    "args": {
+      "key": "process.arguments",
+      "object": "proc",
+      "transformer": "ToLowercaseArray"
+    },
+    "pid": {
+      "key": "process.pid",
+      "object": "proc"
+    },
+    "execFilePath": {
+      "key": "process.cwd",
+      "object": "proc"
+    },
+    "time": {
+      "key": "process.created",
+      "object": "proc"
+    },
+    "id": {
+      "key": "process.extensions.x-rhacs-process-ext.x_unique_id",
```

Comment:
as discussed, this needs be taken out of the extensions. 
`"key": "process.x_unique_id"`

Reply:
updated as suggested.